categories/categories.go

```diff
@@ -0,0 +1,50 @@
+package categories
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/stored_requests"
+)
+
+type Categories struct {
+	categories map[string]map[string]map[string]string
+}
+
+func NewCategories(categoriesFetcher stored_requests.Fetcher) (categories Categories, err error) {
+	categoriesdData := make(map[string]map[string]map[string]string)
+	for k, v := range categoriesFetcher.FetchCategories() {
+
+		cat := make(map[string]map[string]string)
+		for key, value := range v {
+			tmp := make(map[string]string)
+
+			if err := json.Unmarshal(value, &tmp); err != nil {
+				return Categories{
```

Comment:
I think one invalid category mapping file shouldn't prevent all of them from being available. We should just log an error here and skip this mapping file instead of returning an error and no mapping data at all.